test/artifact_test.js

```diff
@@ -70,6 +152,224 @@ suite('Artifacts', function() {
   };
   this.timeout(3 * 60 * 1000);
 
+  suite.only('Blob Storage Type', () => {
+    let tmpobj = tmp.fileSync();
```

Comment:
sure.  that doesn't ensure safe generation, but it's also not so big of a deal in this case if it's not safely generated.